assets/js/modules/optimize/common/UseSnippetInstructions.js

```diff
@@ -40,9 +41,30 @@ export default function UseSnippetInstructions() {
 	const analyticsUseSnippet = useSelect( ( select ) => select( MODULES_ANALYTICS ).getUseSnippet() );
 	const gtmActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'tagmanager' ) );
 	const gtmUseSnippet = useSelect( ( select ) => select( MODULES_TAGMANAGER ).getUseSnippet() );
+	const settingsURL = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-settings' ) );
 
 	if ( ! analyticsActive ) {
-		return null;
+		return (
+			<Fragment>
+				<p>
+					{ __( 'Google Analytics must be active to use Optimize', 'google-site-kit' ) }
+				</p>
+				<p
+					dangerouslySetInnerHTML={ sanitizeHTML(
+						sprintf(
+							'<a href="%1$s">%2$s</a> %3$s',
+							`${ settingsURL }#connect`,
+							_x( 'Click here', 'from "Click here to connect Google Analytics"', 'google-site-kit' ),
+							_x( 'to connect Google Analytics', 'from "Click here to connect Google Analytics"', 'google-site-kit' )
```

Comment:
This also needs to be one translation string - translation strings must never be concatenated, they always need to be full logical sentences in a single string for them to be correctly translatable.